packages/services/src/view-service/auctions.ts

```diff
@@ -33,6 +33,8 @@ const getBech32mAuctionId = (
   return captureGroups.auctionId;
 };
 
+const isInactive = (seqNum?: bigint) => (seqNum === undefined ? false : seqNum > 0n);
```

Comment:
why would the sequence number ever be undefined?

Reply:
I made it optional since `value` (used on line 61 below) may have an undefined `seqNum` property if we haven't saved it yet for some reason.